components/List.css

```diff
@@ -0,0 +1,75 @@
+.component-list{
+    margin-top: 2em;
+}
+
+.List-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+}
+
+.List-wrapper div{
+    width: 200px;
+    
+}
+
+ .order-list, .unordered-list{
+     margin-left: 1em;
+ }
+
+.type-A{
+    list-style-type: upper-alpha;
+}
+
+.type-a{
+    list-style-type: lower-alpha;
+}
+
+.upper-roman{
+    list-style-type: upper-roman;
+}
+
+.lower-roman{
+    list-style-type: lower-roman;
+}
+
+.type-circle{
+    list-style-type:circle;
+}
+
+.type-square{
+    list-style-type: square;
+}
+
+.stacked-list-wrapper{
```

Comment:
Suggestion: you can have a utility class of flex positioning as here it is added for the wrapper. So that you can reuse instead of repeat. You can just customise the same class with your own styles.

Reply:
Everything else looks good

Reply:
okay, I'll try to fix all issues in refactor branch. Thank you for your review.